model/src/main/java/org/mskcc/cmo/metadb/model/SampleMetadata.java

```diff
@@ -329,6 +356,18 @@ public void setCmoSampleIdFields(Map<String, String> cmoSampleIdFields) {
         this.cmoSampleIdFields = cmoSampleIdFields;
     }
 
+    public Map<String, String> getAdditionalProperties() {
+        return additionalProperties;
+    }
+
+    public void setAdditionalProperties(Map<String, String> additionalProperties) {
+        this.additionalProperties = additionalProperties;
+    }
+
+    public void addAdditionalProperty(String property, String value) {
```

Comment:
add this if (additionalProperties == null) {
            additionalProperties = new ArrayList<>();
        }

Reply:
these are initialized by default [here](https://github.com/mskcc/cmo-metadb/pull/529/files#diff-aac420975676b97f92aa9af2e9faa58e68fdce36d332e14b80d80f73796ac64eR87)